Add getStaticProps override exportPathMap (#31906)
vercel/next.js: docs/advanced-features/static-html-export.md

## Caveats

> - With `next export`, we build an HTML version of your app. At export time, we call [`getStaticProps`](/docs/basic-features/data-fetching.md#getstaticprops-static-generation) for each page that exports it, and pass the result to the page's component.

Reviewer: ```suggestion
- At export time, [`getStaticProps`](/docs/basic-features/data-fetching.md#getstaticprops-static-generation) is called for each page that exports it, and the result passed to the page's component. This creates an HTML version of your app.
```
